ocaml/idl/datamodel.ml

```diff
@@ -3442,6 +3442,10 @@ let vdi_resize_online = call
     ~name:"resize_online"
     ~in_oss_since:None
     ~in_product_since:rel_rio
```

Comment:
I've just realized that we can remove `~in_product_since:rel_rio`, because that's already specified by the `~lifecycle`: https://github.com/xapi-project/xen-api/blob/master/ocaml/idl/datamodel.ml#L3687